index.js

```diff
@@ -124,6 +124,8 @@ if (dryRun) {
 const cutoffDate = moment().subtract(numDaysOld, 'd').format('YYYY-MM-DD');
 console.log(`Getting issues that haven't been touched since ${cutoffDate}`);
 
+var pickedIssues;
```

Comment:
(I see in a later commit you changed it to `const pickedIssues = [];`, that works too.